packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -40,7 +73,7 @@ MeetingUtil.joinMeeting = (meeting) => {
         throw err;
       });
   }
-  return Promise.reject();
+  return Promise.reject(new Error('You need a meeting object.'));
```

Comment:
Same comment as above

Reply:
so weird, i made these changes last night, must have not committed